cmake/Modules/LocalJpeg.cmake

```diff
@@ -29,6 +34,10 @@ else()
     #   Manually-specified variables were not used by the project:
     #
     #     WITH_CRT_DLL
+    #
+    # TODO(wtc): Delete one of -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} and
+    # -DCMAKE_BUILD_TYPE=Release.
```

Comment:
See line 49 and line 57.

Reply:
go with release, that is what we do for dav1d, rav1e, svt